package.json

```diff
@@ -69,7 +69,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --pattern '**/*.{ts,js,tsx,jsx,scss,css,html,md}' --check --staged"
+      "pre-commit": "pretty-quick --pattern '**/*.{ts,js,tsx,jsx,scss,css,html,md,json}' --check --staged"
```

Comment:
@elikatsis, if you change it to  `"pre-commit": "npm run format:check:staged"` will it work?

Reply:
It still looks for `yarn` in `PATH`.

It does work:
```
husky > pre-commit (node v12.16.1)
npm WARN lifecycle The node binary used for scripts is
/tmp/yarn--1582820969914-0.2191866932481028/node but npm is using
/usr/local/bin/node itself. Use the `--scripts-prepend-node-path` option to
include the path for the node binary npm was executed with.

> kubeflow-kale-launcher@1.4.0 format:check:staged /home/elias/git/hub/jupyterlab-kubeflow-kale
> pretty-quick --pattern '**/*.{ts,js,tsx,jsx,scss,css,html,md,json}' --check --staged
```

But, it's similar to https://github.com/kubeflow-kale/jupyterlab-kubeflow-kale/pull/39#discussion_r385221434. It would require a local/global installation of `npm`

Reply:
Wouldn't it?